### PR TITLE
Insight cleanup

### PR DESCRIPTION
**Pull request by jburel**
Fix all warning noticed when building the javadoc

To test:
 * make sure you do not see any warning when the javadoc for insight is built.

cc @sbesson

### CODE REVIEW

**Comment by sbesson** [comment]
From https://ci.openmicroscopy.org/view/Failing/job/OMERO-DEV-breaking-build/ICE=3.5,jdk=8_LATEST,label=trout/457/consoleFull

```
22:01:49 Constructing Javadoc information...
22:01:54 Standard Doclet version 1.8.0_51
22:01:54 Building tree for all the packages and classes...
22:03:14 Building index for all the packages and classes...
22:03:16 Building index for all classes...
22:03:16 Building jar: /opt/hudson/workspace/OMERO-DEV-breaking-build/src/components/insight/target/insight-javadoc.jar
```

Removing breaking label.

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [OMERO-DEV-merge-push#67](https://ci.openmicroscopy.org/job/OMERO-DEV-merge-push/67/). See the [console output](https://ci.openmicroscopy.org/job/OMERO-DEV-merge-push/67/consoleText) for more details.
Possible conflicts:
  - PR #4044 jburel 'Roi annotations'
    - components/insight/SRC/org/openmicroscopy/shoola/util/roi/model/ROIShape.java
    - components/insight/SRC/org/openmicroscopy/shoola/agents/measurement/util/model/FigureTableModel.java
    - components/insight/SRC/org/openmicroscopy/shoola/env/data/views/calls/ROISaver.java
  - PR #4051 dominikl 'Add ability to run scripts against fa's from OMERO.Insight'
    - components/insight/SRC/org/openmicroscopy/shoola/env/data/model/ScriptObject.java
  - PR #4055 dominikl 'Rendering settings fixes'
    - components/insight/SRC/org/openmicroscopy/shoola/env/data/views/calls/RenderingSettingsSaver.java
  - PR #4083 dominikl 'ROI tool OutOfMemory'
    - components/insight/SRC/org/openmicroscopy/shoola/env/rnd/roi/ROIAnalyser.java

**Comment by jburel** [comment]
Not surprised that it conflicts with few PRs.
Probably need to have that PR in soon so we can move forward.
Many files have been touched 
cc @dominikl

File: components/insight/SRC/org/openmicroscopy/shoola/env/data/views/calls/SwitchUserGroupLoader.java [hunk 1]
```diff
@@ -47,61 +40,55 @@
...
-	extends BatchCallTree
+public class SwitchUserGroupLoader
+extends BatchCallTree
 {
 
-	/** The experimenter to handle. */
-	private ExperimenterData experimenter;
-	
-	/** The identifier of the group. */
-	private long groupID;
```
Comment (sbesson) [inline components/insight/SRC/org/openmicroscopy/shoola/env/data/views/calls/SwitchUserGroupLoader.java]: This private field is removed by this PR. Is this expected?

Comment (jburel) [inline components/insight/SRC/org/openmicroscopy/shoola/env/data/views/calls/SwitchUserGroupLoader.java]: it is.

**Comment by sbesson** [comment]
Barring one comment, everything looks fine (reviewd with https://github.com/openmicroscopy/openmicroscopy/pull/4092/files?diff=unified&w=1). As discussed with @jburel and @dominikl, understanding the consequences of this PR for rebases, the next step is to get this PR merged as soon as possible and resolve conflicts in the other PRs reported in https://github.com/openmicroscopy/openmicroscopy/pull/4092#issuecomment-134796727.

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [OMERO-DEV-merge-push#68](https://ci.openmicroscopy.org/job/OMERO-DEV-merge-push/68/). See the [console output](https://ci.openmicroscopy.org/job/OMERO-DEV-merge-push/68/consoleText) for more details.
Possible conflicts:
  - PR #4044 jburel 'Roi annotations'
    - components/insight/SRC/org/openmicroscopy/shoola/util/roi/model/ROIShape.java
    - components/insight/SRC/org/openmicroscopy/shoola/agents/measurement/util/model/FigureTableModel.java
    - components/insight/SRC/org/openmicroscopy/shoola/env/data/views/calls/ROISaver.java

**Comment by sbesson** [comment]
From https://ci.openmicroscopy.org/job/OMERO-DEV-merge-build/ICE=3.5,jdk=8_LATEST,label=octopus/71/consoleFull

```
13:17:16 Constructing Javadoc information...
13:17:19 Standard Doclet version 1.8.0_51
13:17:19 Building tree for all the packages and classes...
13:18:33 Building index for all the packages and classes...
13:18:34 Building index for all classes...
13:18:34 Building jar: /opt/hudson/workspace/OMERO-DEV-merge-build/src/components/insight/target/insight-javadoc.jar
```

assuming we understand the impact of this PR on future rebases, any veto to getting this merged now @jburel @joshmoore ?

**Comment by jburel** [comment]
rebase will be difficult so possible PR to rebase will have to be "simple"
cc @dominikl

**Comment by jburel** [comment]
--no-rebase

**Pull request merged**